content/news/playtest-20151031.md

```diff
@@ -0,0 +1,173 @@
+<p markdown="1">
+Halloween is upon us, and the latest OpenRA playtest is full of new tricks and other treats!
+This build includes all of the major new features and balance changes that we have planned for our Christmas release, and it is going to be one of our biggest releases yet!  Check out the
+[full changelog](http://changelog.openra.net) for more details about the balance changes and new
```

Comment:
Remember to eventually link to the specific commit for the changelog, so that it keeps pointing to the right changelog when the page changes.